Doc/whatsnew/3.8.rst

```diff
@@ -1077,6 +1077,15 @@ Changes in the C API
 
   (Contributed by Eddie Elizondo in :issue:`35810`.)
 
+* The :c:macro:`Py_DEPRECATED()` macro has been implemented for MSVC.
+  The macro now must be placed before the symbol name.
+
+  Example::
+
+      Py_DEPRECATED(3.8) int Py_OldFunction(void);
```

Comment:
Should not we use `PyAPI_FUNC()`?

Reply:
Sure, but this is just an example, no?

Reply:
Why not use a good example? It seems to me, that `PyAPI_FUNC()` is necessary in all cases.

Reply:
Okay, I will add `PyAPI_FUNC()`.